firefox-ios/firefox-ios-tests/Tests/XCUITests/AddressesTests.swift

```diff
@@ -147,7 +216,7 @@ class AddressesTests: BaseTestCase {
         mozWaitForElementToExist(app.navigationBars[addresses.title])
         app.buttons[addresses.addAddress].tap()
         mozWaitForElementToExist(app.navigationBars[addresses.addAddress])
-        if iPad() {
+        if !app.staticTexts["Name"].exists {
```

Comment:
In which scenario this may or may not appear? Usually having this check could be flaky because the element could not be there, but then it appears.. or are we waiting enough to be sure that we are in the right screen where that element is for sure or is not depending on the scenario? (I see it was an if iPad condition before...)

Reply:
Sometimes the screen layout is not loading, but i noticed this happens only on automation runs, not manually.
I will add a ticket for it after further investigation.